.github/workflows/cross-browser-testing.yml

```diff
@@ -0,0 +1,40 @@
+name: 'BrowserStack Test'
+on: [push, pull_request, workflow_dispatch]
+
+jobs:
+  ubuntu-job:
```

Comment:
I'm curious about why we're calling this `ubuntu-job`. Are we planning on having other jobs for different OSes?

Reply:
Oh interesting. I pulled the entire job from https://github.com/browserstack/github-actions?tab=readme-ov-file#sample-workflow-with-usage-of-both-actions